soc/st/stm32/stm32wbax/hci_if/linklayer_plat_adapt.c

```diff
@@ -207,11 +165,6 @@ void LINKLAYER_PLAT_DisableIRQ(void)
 	irq_counter++;
 }
 
-void LINKLAYER_PLAT_Assert(uint8_t condition)
-{
-	__ASSERT_NO_MSG(condition);
-}
-
```

Comment:
This should be kept.
It allows to generate Zephyr ASSERTS

Reply:
Fixed